resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,172 @@
+{% endhint %}
+
+   1. Verify the push token sync with the Experience Cloud ID service (ECID).  
+   2. To verify, launch your app connected to a Project Griffon session.   
+   3. In the list of events, verify that you have an event with type _SetPushIdentifier_.
+   4. In the details panel on the right, verify the value of the push token for this device.  
+      The value in pushIdentifier is the same value that is sent to the ECID service. For more information, see screenshot   below.  
+      
+![Verify SetPushIdentifier event](../../.gitbook/assets/push_token_to_identity.png "Verify SetPushIdentifier event.")
+
+   5. To verify that app’s push token is mapped to the correct Experience cloud ID(ECID) in the Campaign instance, click  **_Adobe Campaign -> Administrator -> Channels -> Mobile App (AEP SDK)_**.  
+   6. Select your app, under the mobile application subscribers verify that the Experience Cloud ID and the Registration token for the user is displayed.  
+   
+   ![App subscriber list, verify mid and push token](../../.gitbook/assets/subscriber_list_ios.png "App subscriber list, verify mid and push token.")  
+   
+   6. If you are using Charles, verify that the push token has successfully synced with the ECID service.  
+   7. Check for the **demdex request**, which is marked with the red line in the screenshot below.  
+   8. Verify the successful response(200) for this network call.       
+ 
+ ![ECID network request for push token sync](../../.gitbook/assets/charles_demdex_call_ios.png "ECID network request for push token sync.")
+ 
+  {% endtab %}
+  {% endtabs %}
+
+
+ ## Troubleshooting push notification events tracking  
+
+The following events are related to the tracked push notifications:
+  * Impression (Notification is delivered.)
+  * Click (User clicked the notification.)
+  * Open (App opened in response to user's click on notification.)  
+  For more information about implementing the tracking of push notification events, see [Push Tracking](https://helpx.adobe.com/campaign/kb/push-tracking.html).  
+  
+###  Troubleshooting using Project Griffon  
+
+  You can verify the push token sync with the ECID service in Project Griffon.  
+  
+  1. Connect your app to a Griffon session.  
+  2. Send a push notification to app from your Campaign instance.  
+  To know more about how to send push notification, see [Preparing and sending a push notification](https://docs.adobe.com/content/help/en/campaign-standard/using/communication-channels/push-notifications/preparing-and-sending-a-push-notification.html).
+  3. Receive a push notification in the device.  
+  4. Click on the push notification to launch the app.  
+  5. In the list of events, verify that you have an event with type CollectData.  
+  6. In the Details panel on the right, verify the value of the action.  
+  The value of the action should be 7 for impression, 2 for the click, and 1 for open.  
+  It is shown in the screenshots below.  
```

Comment:
Please remover this sentence as it's not necessary. (Users will automatically look at the screenshot. )